{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--e4664ce2-0648-4a2e-a532-6635532e7fb4",
    "created": "2020-05-26T20:29:12.373685Z",
    "modified": "2020-05-26T20:29:12.373685Z",
    "name": "CVE-2015-2317",
    "description": "The utils.http.is_safe_url function in Django before 1.4.20, 1.5.x, 1.6.x before 1.6.11, 1.7.x before 1.7.7, and 1.8.x before 1.8c1 does not properly validate URLs, which allows remote attackers to conduct cross-site scripting (XSS) attacks via a control character in a URL, as demonstrated by a \\x08javascript: URL."
}